packages/documentation-framework/routes.js

```diff
@@ -50,7 +50,8 @@ const sourceOrder = {
   react: 1,
   'react-next': 1.1,
   'react-composable': 1.2,
-  'react-legacy': 1.3,
+  'react-deprecated': 1.3,
+  'react-legacy': 1.4,
```

Comment:
Do we still have plans to use `react-legacy` after this change and introduction of `react-deprecated`?

Reply:
ultimately react-deprecated will replace the legacy here so I can remove it, but probably not until it's been removed after the breaking change. I don't want to remove it before then.
The deprecated work is going in the v5 branch

Reply:
Got it, so maybe we make a follow-up issue to remove this when we're ready?